packages/react/src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -485,13 +485,13 @@ exports[`Button renders DefaultButton correctly 1`] = `
           font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
           font-size: 14px;
           font-weight: 400;
-          height: 32px;
+          min-height: 32px;
           min-width: 80px;
           outline: transparent;
-          padding-bottom: 0;
+          padding-bottom: 6px;
```

Comment:
@khmakoto I think you need to regenerate the snapshots after your change to remove the 6px padding.

Reply:
Yup, just sent that update, thanks for the reminder!